hivemind_listener/protocol.py

```
@@ -0,0 +1,484 @@
+from ovos_plugin_manager.templates.transformers import AudioLanguageDetector
+from ovos_plugin_manager.templates.tts import TTS
+from ovos_plugin_manager.templates.vad import VADEngine
+from ovos_plugin_manager.tts import OVOSTTSFactory
+from ovos_plugin_manager.vad import OVOSVADFactory
+from ovos_plugin_manager.wakewords import OVOSWakeWordFactory
+from ovos_simple_listener import SimpleListener, ListenerCallbacks
+from ovos_utils.fakebus import FakeBus
+from ovos_utils.log import LOG
+
+from hivemind_bus_client.message import HiveMessage, HiveMessageType, HiveMindBinaryPayloadType
+from hivemind_core.agents import OVOSProtocol
```

Ruff (0.8.2)

`hivemind_core.agents.OVOSProtocol` imported but unused

Remove unused import: `hivemind_core.agents.OVOSProtocol`

(F401)